frontend/packages/topology/src/utils/topology-utils.ts

```diff
@@ -97,6 +98,7 @@ export const getResourceKind = (node: Node): K8sResourceKindReference => {
 };
 
 export const updateTopologyResourceApplication = (
+  t: TFunction,
```

Comment:
Again it seems we're passing around `t` to localize the return value of async operations that won't be re-run if the locale changes.
Seems unnecessary to pass `t` around everywhere for this and instead use the global `i18next`